tensorflow/lite/micro/micro_allocator_test.cc

```diff
@@ -795,4 +795,71 @@ TF_LITE_MICRO_TEST(TestOperatorInputsNotInSubgraphInputs) {
   TF_LITE_MICRO_EXPECT_EQ(0, eval_tensors[4].data.uint8 - start);
 }
 
+TF_LITE_MICRO_TEST(TestTypicalFirstOpAndSecondOpWithScratchTensors) {
+  constexpr int nbr_tensors = 6;
+  tflite::AllOpsResolver op_resolver = tflite::testing::GetOpResolver();
+  tflite::NodeAndRegistration* node_and_registration;
+  const int32_t metadata_buffer[tflite::testing::kOfflinePlannerHeaderSize +
+                                nbr_tensors] = {
+      1, 0, nbr_tensors,  // header: version, subgraph, nbr tensors
```

Comment:
Google style guide generally suggests having inline comments like this:

```c++
  const int32_t metadata_buffer[tflite::testing::kOfflinePlannerHeaderSize +
                                nbr_tensors] = {/*version=*/1,
                                                /*subgraph=*/0,
                                                number_tensors,
                                                /*t0=*/0,
                                                /*t1=*/0,
                                                /*t2=*/0,
                                                /*t3=*/48,
                                                /*t4=*/-1};
```

Reply:
Done